### PR TITLE
Added schema URL for all packages lower than 1.0.0

### PR DESCRIPTION
**Pull request by weslenteche**
Added schema URL inclusion for all packages listed below that are in a version lower than 1.0.0.

- [`open-telemetry/opentelemetry-auto-cakephp`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-cakephp) v0.0.1
- [`open-telemetry/opentelemetry-auto-codeigniter`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-codeigniter) v0.0.7
- [`open-telemetry/opentelemetry-auto-ext-amqp`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-ext-amqp) v0.0.3
- [`open-telemetry/opentelemetry-auto-guzzle`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-guzzle) v0.0.2
- [`open-telemetry/opentelemetry-auto-io`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-io) v.0.0.11
- [`open-telemetry/opentelemetry-auto-laravel`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-laravel) v.0.0.25
- [`open-telemetry/opentelemetry-auto-mongodb`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-mongodb) v.0.0.5
- [`open-telemetry/opentelemetry-auto-openai-php`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-openai-php) v.0.0.1
- [`open-telemetry/opentelemetry-auto-pdo`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-pdo) v0.0.13
- [`open-telemetry/opentelemetry-auto-psr3`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-psr3) v.0.0.6
- [`open-telemetry/opentelemetry-auto-psr6`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-psr6) v.0.0.2
- [`open-telemetry/opentelemetry-auto-psr14`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-psr14) v.0.0.2
- [`open-telemetry/opentelemetry-auto-psr16`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-psr16) v.0.0.2
- [`open-telemetry/opentelemetry-auto-wordpress`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-wordpress) v.0.0.15
- [`open-telemetry/opentelemetry-auto-yii`](https://packagist.org/packages/open-telemetry/opentelemetry-auto-yii) v.0.0.4

### CODE REVIEW

File: src/Instrumentation/Laravel/src/LaravelInstrumentation.php [hunk 1]
```diff
@@ -13,7 +13,11 @@ class LaravelInstrumentation
 
     public static function register(): void
     {
-        $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.laravel');
+        $instrumentation = new CachedInstrumentation(
+            'io.opentelemetry.contrib.php.laravel',
+            null,
+            'https://opentelemetry.io/schemas/1.25.0'
```
Comment (ChrisLightfootWild) [inline src/Instrumentation/Laravel/src/LaravelInstrumentation.php:19]: `1.24.0` is the max supported here.

Comment (ChrisLightfootWild) [reply src/Instrumentation/Laravel/src/LaravelInstrumentation.php:19]: Apologies, re-reading that and I didn't give much info!

It's been discussed to aim to release `0.x` packages with a dependency on `php: ^8.0` (as auto-instrumentation extension works with that).

`sem-conv: 1.24.0` was the last version before the dependency in the package was bumped to `php: ^8.1`.

Hope this helps explain my previous comment 👍

Comment (weslenteche) [reply src/Instrumentation/Laravel/src/LaravelInstrumentation.php:19]: I had understood in [this comment](https://github.com/open-telemetry/opentelemetry-php/issues/1312#issuecomment-2118658228) that we should update to the latest semantic version which is currently 1.25.

For example, the contrib-auto-slim package has a version above `>= 1.0.0` and in this [PR ](https://github.com/open-telemetry/opentelemetry-php-contrib/pull/259) it was defined to use the URL schema in version 1.25, however in composer.json it has the `php: ^8.0`.

I was a little confused regarding whether the URL schema version should be defined based on the supported PHP versions according to their respective composer.json or go with the latest published version

**Comment by brettmc** [comment]
> I had understood in https://github.com/open-telemetry/opentelemetry-php/issues/1312#issuecomment-2118658228 that we should update to the latest semantic version which is currently 1.25.

I did say latest in that comment, but in a subsequent discussion (it might have been at SIG), we decided that we should favour getting a 1.0 release of all auto-instrumentation modules with 8.0 support, if possible. Going with semconv 1.25 would block 8.0.

**Comment by weslenteche** [comment]
> I did say latest in that comment, but in a subsequent discussion (it might have been at SIG), we decided that we should favour getting a 1.0 release of all auto-instrumentation modules with 8.0 support, if possible. Going with semconv 1.25 would block 8.0.

Thank you for clarifying my doubt

**Comment by brettmc** [comment]
LGTM :+1:

**Review by brettmc — APPROVED** [review]

**Review by ChrisLightfootWild — APPROVED** [review]

**Pull request merged**